task-3/task3.go

```diff
@@ -0,0 +1,23 @@
+package main
+import "fmt"
+
+func main(){
+
+	var n int
+
+	fmt.Scanf("%d", &n)
+	nums:=make([]int, n)
+
+	for i:=0; i<n; i++ {
+		fmt.Scanf("%d",&nums[i])
+	}
+	end_el:=nums[n-1]
+	for i:=n - 1; i>0;i--{
+		nums[i] = nums[i-1]
+	} 
+	nums[0] = end_el
```

Comment:
в Go принято использовать camelCase

```suggestion
	lastElem := nums[n-1]
	for i := n - 1; i > 0; i-- {
		nums[i] = nums[i-1]
	} 
	nums[0] = lastElem
```